packages/sdk-components-react/src/theme/tailwind-classes.ts

```diff
@@ -0,0 +1,203 @@
+/**
+ * Quik and dirty implementation of tailwind classes conversion to webstudio styles.
+ */
+import type { EmbedTemplateStyleDecl } from "@webstudio-is/react-sdk";
+import * as variables from "./radix-variables";
+import { theme } from "./tailwind-theme";
+import { parseCssValue } from "@webstudio-is/css-data";
+
+// https://github.com/tailwindlabs/tailwindcss/blob/master/src/css/preflight.css
+export const preflight = (): EmbedTemplateStyleDecl[] => [
```

Comment:
we already have our normalize and reset ... is this not the same? how can we avoid having both?

Reply:
yeah, better move, otherwise this looks like a place for all sorts of resets, but also check if we don't some of it already in our preset/normalize

Reply:
moved